Readme.md

```diff
@@ -15,7 +15,7 @@ Please read the [FAQ](https://dolphin-emu.org/docs/faq/) before using Dolphin.
 * OS
     * Windows (10 or higher).
     * Linux.
-    * macOS (10.13 High Sierra or higher).
+    * macOS (10.14 High Sierra or higher).
```

Comment:
This should be changed from High Sierra to Mojave.

Reply:
whoops thanks